spec/administrate/views/fields/text/_form_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "rails_helper"
+require "administrate/field/text"
+
+describe "fields/text/_form", type: :view do
```

Comment:
Minor thing really, but I generally prefer to avoid the mixin approach for RSpec (and we also don't need to specify the type):

```suggestion
RSpec.describe "fields/text/_form" do
```

Reply:
Cheers, thank you @nickcharlton!